js/index.js

```diff
@@ -0,0 +1,210 @@
+        if (findItem) {
+            findItem.value = entity.value;
+            findItem.check = entity.check;
+        } else {
+            todoItems.push(entity);
+        }
+        lsg.set('todoItems', todoItems);
+    }
+    delete = (idx) => {
+        if(whatType(idx)==="string")idx = Number(idx);
+        let todoItems = lsg.get("todoItems");
+        let newTodoItems = [];
+        if (todoItems.length > 0) {
+            for (const todoItem of todoItems) {
+                if (todoItem.idx !== idx) {
+                    newTodoItems.push(todoItem);
+                }
+            }
+        }
+        lsg.set('todoItems', newTodoItems);
+    }
+
+    //LocalStorage확인 후 Load하는 함수
+    build = () => {
+        let todoItems = lsg.get("todoItems");
+        if (todoItems.length) {
+            for (const item of todoItems) {
+                this.$todoList.append(this.getFrame(item.idx, item.value, item.check));
+            }
+            this.updateCount();
+        }
+    }
+
+    //이벤트 등록
+    eventRegist = () => {
+        this.$todoInput.addEventListener('keyup', this.onAddInput);
+        this.$todoFooter.addEventListener('click', this.switchTap);
+        this.$todoList.addEventListener('click', this.onChangeToggle);
+        this.$todoList.addEventListener('dblclick', this.onEditingByDblClick);
+        this.$todoList.addEventListener('keyup', this.onEditingByKeyup);
+    }
+
+    //수정적용 및 취소
+    onEditingByKeyup = ({ target, key }) => {
+        if (!target) return;
+        const close = () => target.parentElement.classList.remove('editing');
+        if (target.tagName === "INPUT" && key === "Escape") {
+            target.parentElement.classList.remove('editing');
+            target.value = this.previous;
+            close();
+        } else if (target.tagName === "INPUT" && key === "Enter") {
+            //TODO 공백으로 만든 뒤 엔터를 칠 경우의 로직 구현 필요
+            target.previousElementSibling.querySelector(".label").textContent = target.value;
+            this.save(this.getTodoEntity(target.parentElement.dataset["idx"]));
+            close();
+        }
+        this.updateCount();
+    }
+
+    //double click 이벤트 - 수정하기
+    onEditingByDblClick = ({ target }) => {
+        if (!target) return;
+        if (target.tagName === "LABEL") {
+            this.previous = target.textContent;
+            target.parentElement.parentElement.classList.add('editing');
+        }
+        this.updateCount();
+    }
+
+    //todoItems 완료 및 삭제
+    onChangeToggle = ({ target }) => {
+        if (!target) return;
+        const cls = target.classList;
+        if (cls.contains("toggle")) {
+            let findTarget = target.parentElement.parentElement;
+            findTarget.classList.toggle('completed');
+            this.save(this.getTodoEntity(findTarget.dataset['idx']));
+            let currentTap = this.$todoFooter.querySelector(".selected");
+            currentTap.click();
+        } else if (cls.contains("destroy")) {
```

Comment:
```suggestion
        }
        if (cls.contains("destroy")) {
```
어차피 겹치지 않기 때문에 else는 사용하지 않아도 될 것 같습니다.